securitycenter/api/tests/SecurityCenterTests.cs

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Xunit;
+using Snippets;
+
+public class SecurityCenterTests

```

Comment:
I believe some of the code here would be better off in a Fixture (in case you are not familiar with XUnit fixtures, here's the [link](https://xunit.net/docs/shared-context)).
Also, see #1020 for a good Fixture implementation. Ping me if I can help.

Reply:
Added a text fixture with 3 environment variables. Not sure if the Fixture could be/should be doing more.

Reply:
Addressed everything minus setting up a Topic and tearing it down. Our other code samples use an ACL'd organization exclusively used to host code samples. I think it'd be alright to keep as. LMK your thoughts.